### DISCUSSION TITLE
Brita-Workshop Compatibility Patch for CEC version

### DISCUSSION THREAD

**Discussion by aquelarrefox** [DevoTan/the-workshop#2] [General]
hi i wan to make a mod like Brita-Workshop Compatibility Patch (https://steamcommunity.com/sharedfiles/filedetails/?id=2916562978&searchtext=brita) for Guns of 93, i can chenge the gun id to patch tahat but i want to work with The Workshop Collection (CEC) instead of the old branches. could you tell me what i have to change in the brita version (its not mine but im also interesting in it) to make it work with the CEC version or the recipes will work fine with no change.

thanks for your time.

**Comment by DevoTan** [comment]
If you'd like to do it you may, but I'd reach out to djvirus on discord and ask for his permission. I can't stop you if you don't since I'm in no position to enforce his copyright though.

**Comment by aquelarrefox** [comment]
the mod i mention its DarkJarris, i want to know what its need to make thar patch for work with the CEC version

**Comment by DevoTan** [comment]
Looking over the script, all you'd have to do is remove `keep TW.MetalWorkbench,` and add `NearItem:Armory Table,` under `NeedToBeLearn:true,` on both `recipe Disassemble Gun` and `recipe Disassemble Gun Attachment` in the `DJ_BritaPatch.txt` script file.
I will mention because this a mounted mod which does not use any of the assets from Brita's or The Workshop the author, DarkJarris, is fully within his right to file a DMCA takedown request against any re-uploads of his mod without his approval as he states in the description of his mod, "You may NOT copy, redistribute, re-upload translations, or otherwise tamper with this mod."
